sharding/nodesCoordinator/sovereignNumberOfShardsComputer_test.go

```diff
@@ -0,0 +1,56 @@
+package nodesCoordinator
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSovereignIndexHashedNodesCoordinator_ComputeNumberOfShards(t *testing.T) {
+	t.Parallel()
+
+	t.Run("should not work with 0", func(t *testing.T) {
+		t.Parallel()
+
+		ssc := NewSovereignNumberOfShardsComputer()
+		eligibleMap := make(map[uint32][]Validator)
+		nodesConfig := &epochNodesConfig{
+			eligibleMap: eligibleMap,
+		}
+
+		nbShards, err := ssc.ComputeNumberOfShards(nodesConfig)
+		require.Equal(t, ErrInvalidNumberOfShards, err)
+		require.Equal(t, uint32(0), nbShards)
+	})
+
+	t.Run("should work only with 1", func(t *testing.T) {
+		t.Parallel()
+
+		ssc := NewSovereignNumberOfShardsComputer()
+		eligibleMap := make(map[uint32][]Validator)
+		eligibleMap[0] = []Validator{&validator{}}
```

Comment:
Can we use in these tests `core.SovereignChainID` instead of 0?

Reply:
fixed